src/main/java/org/opentripplanner/routing/impl/OrcaFareServiceImpl.java

```diff
@@ -0,0 +1,404 @@
+                    return RideType.COMM_TRANS_LOCAL_SWIFT;
+                }
+            }
+        );
+        classificationStrategy.put(
+            KC_METRO_AGENCY_ID,
+            routeData -> {
+                if (routeData.getType() == ROUTE_TYPE_FERRY &&
+                    routeData.getLongName().contains("Water Taxi: West Seattle")) {
+                    return RideType.KC_WATER_TAXI_WEST_SEATTLE;
+                } else if (routeData.getType() == ROUTE_TYPE_FERRY &&
+                    routeData.getDesc().contains("Water Taxi: Vashon Island")) {
+                    return RideType.KC_WATER_TAXI_VASHON_ISLAND;
+                }
+                return RideType.KC_METRO;
+            }
+        );
+        classificationStrategy.put(SOUND_TRANSIT_AGENCY_ID, routeData -> RideType.SOUND_TRANSIT);
+        classificationStrategy.put(EVERETT_TRANSIT_AGENCY_ID, routeData -> RideType.EVERETT_TRANSIT);
+        classificationStrategy.put(PIERCE_COUNTY_TRANSIT_AGENCY_ID, routeData -> RideType.PIERCE_COUNTY_TRANSIT);
+        classificationStrategy.put(SKAGIT_TRANSIT_AGENCY_ID, routeData -> RideType.SKAGIT_TRANSIT);
+        classificationStrategy.put(SEATTLE_STREET_CAR_AGENCY_ID, routeData -> RideType.SEATTLE_STREET_CAR);
+        classificationStrategy.put(WASHINGTON_STATE_FERRIES_AGENCY_ID, routeData -> RideType.WASHINGTON_STATE_FERRIES);
+        classificationStrategy.put(KITSAP_TRANSIT_AGENCY_ID, routeData -> RideType.KITSAP_TRANSIT);
+
+        washingtonStateFerriesFares.put(
+            "Seattle - Bainbridge Island",
+            ImmutableMap.of(Fare.FareType.regular, 9.05f, Fare.FareType.youth, 4.50f, Fare.FareType.senior, 4.50f)
+        );
+        washingtonStateFerriesFares.put(
+            "Seattle - Bremerton",
+            ImmutableMap.of(Fare.FareType.regular, 9.05f, Fare.FareType.youth, 4.50f, Fare.FareType.senior, 4.50f)
+        );
+        washingtonStateFerriesFares.put(
+            "Mukilteo - Clinton",
+            ImmutableMap.of(Fare.FareType.regular, 5.55f, Fare.FareType.youth, 2.75f, Fare.FareType.senior, 2.75f)
+        );
+        washingtonStateFerriesFares.put(
+            "Fauntleroy - Vashon Island",
+            ImmutableMap.of(Fare.FareType.regular, 5.95f, Fare.FareType.youth, 2.95f, Fare.FareType.senior, 2.95f)
+        );
+        washingtonStateFerriesFares.put(
+            "Fauntleroy - Southworth",
+            ImmutableMap.of(Fare.FareType.regular, 7.10f, Fare.FareType.youth, 3.55f, Fare.FareType.senior, 3.55f)
+        );
+        washingtonStateFerriesFares.put(
+            "Edmonds - Kingston",
+            ImmutableMap.of(Fare.FareType.regular, 9.05f, Fare.FareType.youth, 4.50f, Fare.FareType.senior, 4.50f)
+        );
+        washingtonStateFerriesFares.put(
+            "Point Defiance - Tahlequah",
+            ImmutableMap.of(Fare.FareType.regular, 5.95f, Fare.FareType.youth, 2.95f, Fare.FareType.senior, 2.95f)
+        );
```

Comment:
Missing info for the following routes:

- `Anacortes - Friday Harbor`
- `Anacortes - Lopez Island`
- `Anacortes - Orcas Island`
- `Anacortes - Shaw Island`
- `Coupeville  - Port Townsend`
- `Port Townsend - Coupeville`
- `Southworth - Vashon Island`

A lot of these are not very close to Community Transit's service area, but they are missing. Maybe check in with @jon-campbell-ibigroup if we can just add a comment if they aren't needed.

Reply:
@evansiroky where did you get this information from? I'm working from the spreadsheet provided by Jon: https://docs.google.com/spreadsheets/d/1SgADItHr4b8W9pqAeYJaSLnPZXN-IowXOwql9pVTX_8/edit#gid=771305337. This only contains the discounts I have already included. If these are required (and I'll check with Jon) can you please provide the URL so I can get the correct fares. Thanks.

Reply:
I double-checked on each transit agency's website. For WSF I checked every possibility via [their fares page](https://www.wsdot.wa.gov/ferries/fares/). Also, I grew up in Seattle, so I kind of have some local knowledge of the fare system, so I knew that there these other routes that should have fares, so I just went ahead and double-checked it all.